fw-child/template/query/variable-item.php

```diff
@@ -11,7 +11,7 @@
 }
 ?>
 
-<div class="card position-relative variable-item" id="v-<?php echo esc_attr( $variable_slug ); ?>">
+<div class="card position-relative variable-item no-resize" id="v-<?php echo esc_attr( $variable_slug ); ?>">
```

Comment:
Since cards for variables have a different effect that could still be considered as a "resize", the class `.no-resize` could create confusion about its effect. I suggest a more precise name of `.no-enlarge`.